packages/react-dom/src/events/__tests__/SelectEventPlugin-test.js

```diff
@@ -43,7 +43,6 @@ describe('SelectEventPlugin', () => {
       <input type="text" onMouseDown={function() {}} />,
       container,
     );
-    node.focus();
```

Comment:
Why was this removed?

Reply:
Calling `focus()` on an already focused element does not dispatch a FocusEvent. This was a bug in JSDOM that was fixed in https://github.com/jsdom/jsdom/pull/2700